### DISCUSSION TITLE
Most maps are unavailable currently because archive.org was DDoSed

### DISCUSSION THREAD

**Discussion by LegendDD44** [nzp-team/nzportable#935] [General]
Is there any other way to access these maps?

**Comment by 26lake** [comment]
not at the moment. website is apparently starting to rise back up but no hope atm

**Reply by motolav** [reply]
It should be back up this week, there was a post on their mastodon saying it was days not weeks until it should be back up 
https://mastodon.archive.org/@internetarchive/113290094683712789

**Reply by 26lake** [reply]
ik ik i'm just a little impatient lmao
